File: frontend/src/app/models/TextAudio.ts
export class TextAudio {
  id: number;
  audiostart: number;
  audioend: number;
  text: string;
  fileid: number;
  speaker: string;
  labeled: number;
  correct: number;
  wrong: number;

  constructor(id: number, audiostart: number, audioend: number, text: string, fileid: number, speaker: string, labeled: number, correct: number, wrong: number) {
    this.id = id;
    this.audiostart = audiostart;
    this.audioend = audioend;
    this.text = text;
    this.fileid = fileid;
    this.speaker = speaker;
    this.labeled = labeled;
    this.correct = correct;
    this.wrong = wrong;
  }
}

File: backend/src/main/scala/models/TextAudio.scala
package models

import io.circe.generic.semiauto.{deriveDecoder, deriveEncoder}

case class TextAudio(
                      id: BigInt,
                      audiostart: Float,
                      audioend: Float,
                      text: String,
                      fileid: BigInt,
                      speaker: String,
                      labeled: BigInt,
                      correct: BigInt,
                      wrong: BigInt
                    ) {
}

object TextAudio {
  implicit val encoder = deriveEncoder[TextAudio]
  implicit val decoder = deriveDecoder[TextAudio]
}
